tests/late_joiners.py

```diff
@@ -153,19 +153,19 @@ def run(args):
 
             # check that a new node can catch up after all the requests
             LOG.info("Adding a very late joiner")
-            last_joiner_node = network.create_and_trust_node(
+            late_joiner = network.create_and_trust_node(
                 lib_name=args.package, host="localhost", args=args,
             )
```

Comment:
create_and_trust_node raises on error, it does not return None, the assert is not useful here.